assets/sass/elements/_elements-typography.scss

```diff
@@ -201,10 +201,12 @@ p,
   &:link,
   &:visited,
   &:hover,
+  a#{&}:focus,
```

Comment:
Ah sad, specificity strikes again.

I've seen people double classes up too: '.link-back.link-back:focus'

Thanks for the extra info Jani 👍